build.gradle

```diff
@@ -13,11 +13,30 @@ targetCompatibility = JavaVersion.VERSION_11 // bytecode target version
 
 repositories {
     mavenCentral()
+    maven { url  "https://dl.bintray.com/unit-nva/nva-commons" }
+    maven { url 'https://jitpack.io' }
+}
+
+project.ext{
+    nvaCommonsVersion = '1.0.4'
```

Comment:
It surprises me that jupiterVersion is not in project.ext, but this is not to say that it is incorrect

Reply:
Should i put it there?

Reply:
Why not? :)

Reply:
Moved it to project.ext